src/action_management/include/structs.h

```diff
@@ -0,0 +1,46 @@
+/* In the absence of the Game State team's code, 
+ * here are some theoretical structs containing our needed components.
+ * We use these structs in our implementation of basic action commands.
+*/
+
+
+/* A NPC struct containing:
+ * - dialogue: a line of dialogue NPC will say when spoken to 
+ *
+ */
+struct npc_struct {
```

Comment:
maybe also inventory?

Reply:
```suggestion
struct npc_struct {
```
I thought we weren't handling inventory in our structs?

Reply:
this was just what we thought inventory should have for our functions to work

Reply:
i oh i was referring to the npc_struct maybe it should have some kind of inventory but i guess it's not necessary